### DISCUSSION TITLE
How do i send a message component (buttons, dropdown, etc) through slash commands?

### DISCUSSION THREAD

**Discussion by chrisdewa** [tandemdude/hikari-lightbulb#115] [Q&A] [answered]
I'm looking to make a slash-only bot and I need to send buttons and other components. Are there plans to include them or how do I use them, I haven't found info on this.

**Comment by ghost** [comment, marked as answer]
Components are a hikari feature (not hikari-lightbulb) and were released in the most recent version (2.0.0.dev102 I believe) - you should see their documentation about it, ask in their repo, or join the discord server where you can receive help.
